src/common/components/DisplayExchangedAmount/DisplayExchangedAmount.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { AmountWithCurrencyField } from '@folio/stripes-acq-components';
+import { KeyValue } from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+
+import { useExchangeCalculation } from '../../hooks';
+
+export const DisplayExchangedAmount = ({ currency, exchangeRate, total }) => {
```

Comment:
"**Display**ExchangedAmount" name sounds like a verb. Could you please change the name of the component